edgesec/Makefile

```
@@ -57,18 +57,26 @@ CMAKE_OPTIONS += \
 	-DUSE_NETLINK_SERVICE=OFF \
 	-DUSE_UCI_SERVICE=ON \
 	-DUSE_GENERIC_IP_SERVICE=OFF \
+	-DUSE_RADIUS_SERVICE=ON \
 	-DBUILD_UCI_LIB=OFF \
 	-DBUILD_HOSTAPD=OFF \
 	-DBUILD_SQLITE_LIB=OFF \
 	-DBUILD_PCAP_LIB=OFF \
-	-DBUILD_UUID_LIB=OFF
+	-DBUILD_UUID_LIB=OFF \
+	-DUSE_CLEANER_MIDDLEWARE=ON \
+	-DUSE_HEADER_MIDDLEWARE=ON \
+	-DUSE_PCAP_MIDDLEWARE=ON
```

```suggestion
	-DUSE_CLEANER_MIDDLEWARE=OFF \
	-DUSE_HEADER_MIDDLEWARE=ON \
	-DUSE_PCAP_MIDDLEWARE=OFF
```

I think this config should be the following